Update to the order of the Ukraine page (#4185)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/ukraine.md

    Get support training to teach if you're disabled: "/funding-and-support/if-youre-disabled"
    Teach in England if you trained overseas : "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
    Who do you want to teach? : "/life-as-a-teacher/who-do-you-want-to-teach"

> navigation: 20.90

Author: ```suggestion
navigation: 20.70
```

You can apply for a Skilled Worker visa no matter what type of visa you currently have that allows you to stay in the UK. 

### Safeguarding and criminal records checks

Schools will assess your suitability to teach from a range of different sources, including:

* 2 references from previous employers
* your complete employment history
* evidence of your professional standing in Ukraine – for example, documentation from the [Ministry of Education and Science of Ukraine](https://mon.gov.ua/ua) showing you are qualified and have no sanctions against you as a teacher
* a criminal record check

Teacher training providers will ask for:

* evidence you have a bachelor’s degree from a university (non-UK or in the UK)
* a criminal record check

You can [visit the Ministry of Internal Affairs of Ukraine website](https://vytiah.mvs.gov.ua/app/landing) to find out how to apply for a criminal record check, which will be emailed to you. You can then [contact the Ukrainian Embassy in the UK](https://uk.mfa.gov.ua/en) for a translation of the criminal record check and a letter confirming its validity.

### Your professional qualifications

> You can find out more about [getting a statement of comparability from the UKENIC](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your academic and professional qualifications compare to their English equivalent. However, you'll need scans or photographs of your original documents. 

Reviewer: Should this be UK ENIC?

> You can find out more about [getting a statement of comparability from the UKENIC](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your academic and professional qualifications compare to their English equivalent. However, you'll need scans or photographs of your original documents. 

Author: ```suggestion
You can find out more about [getting a statement of comparability from the UK ENIC](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your academic and professional qualifications compare to their English equivalent. However, you'll need scans or photographs of your original documents. 
```

* help you can get with your application 

If you intend to apply for teacher training, you could consider [finding work as a teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain experience until your teacher training course starts. 

### Fees and financial support

Ukrainian refugees in the UK under a Ukraine visa scheme have access to the same financial support as UK applicants. This means you'll:

> * have the right to [apply for a student loan, bursary, scholarship](/funding-and-support) or an [advanced learner loan](https://www.gov.uk/advanced-learner-loan) to help cover your fees and living expenses

Reviewer: Should the second link be 'apply for an advanced...' to make it consistent for screen readers?

> * have the right to [apply for a student loan, bursary, scholarship](/funding-and-support) or an [advanced learner loan](https://www.gov.uk/advanced-learner-loan) to help cover your fees and living expenses

Author: ```suggestion
* have the right to [apply for a student loan, bursary, scholarship](/funding-and-support) or [apply for an advanced learner loan](https://www.gov.uk/advanced-learner-loan) to help cover your fees and living expenses
```

### Other ways to get teaching work and experience

Supply teachers provide teaching cover when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school. [Find out more about being a supply teacher on the tes website](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher). 

> [Find your local authority (council)](https://www.gov.uk/find-local-council) and then search their website for teaching roles.

Reviewer: i think we should just say 'council' - LA seems internal and GDS says to use council for user-facing services.

> [Find your local authority (council)](https://www.gov.uk/find-local-council) and then search their website for teaching roles.

Author: ```suggestion
[Find your local council](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
```

[Supply teachers](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher) fill in when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school.  

[Find your local authority (council)](https://www.gov.uk/find-local-council) and then search their website for teaching roles.

> You could also consider [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system before you apply for teaching jobs. [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) lists these jobs. 

Author: ```suggestion
You could also consider [working as a teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) or in education support roles. It can be helpful to gain experience of the English education system before you apply for teaching jobs. [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) lists these jobs. 
```
